docs/reference/objects/product/variant/index.md

```diff
@@ -13,6 +13,7 @@ When using a `variant` object you have access to the following attributes:<br>
 # variant.deposit
 
 Returns the [deposit]({% link docs/reference/objects/product/deposit.md %}) of this variant if paying with deposit is enabled, `nil` otherwise.
+If the variant has an active [promotion]({% link docs/reference/objects/product/promotion.md %}) the deposit rate is applied to the promotional price.
```

Comment:
@eballantine reading this actually, I'm a bit puzzled. I don't think the deposit makes any assumption on where it's being applied right? 

I think this statement makes sense but I'm not sure it should be here?
I'd like to see how are you using this to understand a bit better where we might put this information.

Reply:
@geoffreymm ahh yes, you're right to be puzzled, the guidance to a theme developer is that it '_should be_ applied' to promotional price. 
My '_is_ applied' is referring to deposit amount that the customer will be asked to pay at checkout. 
I've updated to reflect, is that clear now or should I maybe include an example/s?

Reply:
@eballantine lovely!! I mean, as a courtesy for the next theme developer, a lil example wouldn't hurt, what do you think?

Reply:
@geoffreymm Only polite, go on then.